modules/system-info/reporters/server.php

```diff
@@ -291,8 +297,8 @@ public function get_mysql_version() {
 
 	/**
 	 * Get write permissions.
-	 *
-	 * Check whether the required folders has writing permissions.
+	 * Check whether the required paths have writing permissions.
+	 * Do not use this function if you want the check write permissions use get_permissions() instead.
```

Comment:
Add deprecation notice instead

Reply:
It's not deprecated, it's just should not be used.

Reply:
But I changed the comment anyway